src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/QueryExecutor.java

```diff
@@ -67,8 +67,13 @@ public Uni<QueryOuterClass.ResultSet> executeWrite(QueryOuterClass.Query query)
     QueryOuterClass.Consistency consistency = queriesConfig.consistency().writes();
     QueryOuterClass.ConsistencyValue.Builder consistencyValue =
         QueryOuterClass.ConsistencyValue.newBuilder().setValue(consistency);
+    QueryOuterClass.Consistency serialConsistency = queriesConfig.serialConsistency();
+    QueryOuterClass.ConsistencyValue.Builder serialConsistencyValue =
+        QueryOuterClass.ConsistencyValue.newBuilder().setValue(serialConsistency);
     QueryOuterClass.QueryParameters.Builder params =
-        QueryOuterClass.QueryParameters.newBuilder().setConsistency(consistencyValue);
+        QueryOuterClass.QueryParameters.newBuilder()
```

Comment:
Do you want to revert the serial consistency change, so it can be done unit test case as suggested in #290 ?